authenticator/src/main/java/com/beyondidentity/authenticator/sdk/AuthView.kt

```diff
@@ -40,23 +40,68 @@ class AuthView : LinearLayout {
     /**
      * Initialize the [AuthView]
      *
-     * @param loginUrl The http endpoint your API backend provides to start the OAUTH2 flow
-     * @param redirectUrl Where should the server redirect after successful authentication with the access code
+     * Use this initialization flow when your backend already has the Oauth configuration setup,
+     * client_id, client_secret, scope, response_type, etc...
```

Comment:
nit: I would say "Use this flow if your backend handles getting the authorization code and exchanging that for an access token and id token. It is up to the caller of this initializer to decide how to invoke their backend in order to initiate the flow, as well as what the response would be. As an example, a response could be a session id associated with the authenticated user or the `userinfo` payload that was queried on the backend."